util/util.go

```diff
@@ -466,22 +466,3 @@ func VerifyTagName(imageSpec string) (types.ImageReference, error) {
 	}
 	return ref, nil
 }
-
-// Cause returns the most underlying error for the provided one. There is a
-// maximum error depth of 100 to avoid endless loops. An additional error log
-// message will be created if this maximum has reached.
-func Cause(err error) (cause error) {
```

Comment:
LGTM just one doubt: Does removing `Cause` breaks public API ?

Reply:
Yes, it does.  I don't see it being used in podman, though.

Reply:
Ah i see. SGTM.